graph/denota/schema.graphql

```diff
@@ -0,0 +1,237 @@
+# TODO look into ID vs Bytes type for id field
+############################################# Nota ###############################################
+type ERC20 @entity {
+  id: ID!
+  isWhitelisted: Boolean
+  # name: String
+	# symbol: String
+	# decimals: Int!
+  # totalSupply: BigInt!
+}
+
+type Transaction @entity {
+  id: ID!
+  timestamp: BigInt!
+  blockNumber: BigInt!
+  transactionHash: String!
+  events: [Event!]! @derivedFrom(field: "transaction")
+}
+
+type Account @entity {
+  id: ID!
+  notasSent: [Nota!]! @derivedFrom(field: "sender")
+  notasReceived: [Nota!]! @derivedFrom(field: "receiver")
+  notasOwned: [Nota!]! @derivedFrom(field: "owner")
+
+  transfersFrom: [Transfer!]! @derivedFrom(field: "from")
+  transfersTo: [Transfer!]! @derivedFrom(field: "to")
+
+  approvalsOwner: [Approval!]! @derivedFrom(field: "owner")
+  approvalsApproved: [Approval!]! @derivedFrom(field: "approved")
+  approvalsForAllOwner: [ApprovalForAll!]! @derivedFrom(field: "owner")
+  approvalsForAllOperator: [ApprovalForAll!]! @derivedFrom(field: "operator")
+	events: [Event!]! @derivedFrom(field: "caller")
+  # TODO: make module specific properties addable by module? 
+  # notasInspected: [Nota!]! @derivedFrom(field: "inspector")
+}
+
+interface Event {
+  id: ID!
+  caller: Account!
+  transaction: Transaction!
+}
+
+type Transfer implements Event @entity(immutable: true) {
+  id: ID!
+  caller: Account! # Can be approved/operator..?
+  from: Account!
+  to: Account!
+  nota: Nota!
+  transaction: Transaction!
+}
+
+type Approval implements Event @entity {
+  id: ID!
+  caller: Account!
+  owner: Account!
+  approved: Account!
+  nota: Nota!
+  transaction: Transaction!
+}
+
+type Nota @entity {
+  id: ID!
+  currency: ERC20
+  escrowed: BigInt
```

Comment:
can you add instant onto the nota? Need an easy way to pull that value in the frontend

Reply:
Hmmm but funded isn't emitted during written right? I need an easy way to pull that value for the nota card for the Direct Send case

Reply:
It's stored in the Written entity as well so can be pulled from there